frontend/packages/erd-core/src/components/ERDRenderer/AppBar/AppBar.tsx

```diff
@@ -39,9 +39,7 @@ export const AppBar: FC = () => {
           <HelpButton />
         </div>
         <ExportButton />
-        <Button variant="solid-primary" size="md">
-          Share
-        </Button>
+        <CopyLinkButton />
```

Comment:
📝 To click `CopyLinkButton`, the active table must be deactivated for now.
Let me fix this next PR.